src/sentry/assistant/guides.py

```diff
@@ -95,4 +95,31 @@
             },
         ]
     },
+    'members': {
+        'id': 3,
+        'cue': _('Tips on inviting your team'),
+        'required_targets': ['member-add'],
+        'steps': [
+            {
+                'title': _('Why invite your team?'),
+                'message': _('Sentry doesn\'t just collect errors. Adding your team lets you '
```

Comment:
There might be an opportunity for more draw here. Triaging and assignment is procedural. Say something about how debugging is hard, collaboration makes it easier and perhaps even fun, and Sentry users are more successful with teammates than without. The vision is of people working together to solve problems.